### PR TITLE
Store isExperimental in local storage.

### PR DESCRIPTION
**Pull request by xPolar**
No issue, discussed in Discord VC.

Store isExperimental in local storage so the debug option in the experimental menu stays consistent on reload.

### CODE REVIEW

**Review by jamiepine — DISMISSED** [review]

File: packages/interface/src/components/device/Stores.tsx [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+const setLocalStorage = (key: string, value: any) =>
+	window.localStorage.setItem(key, JSON.stringify(value));
+
+type NodeState = {
+	isExperimental: boolean;
+	setIsExperimental: (experimental: boolean) => void;
+};
+
+export const useNodeStore = create<NodeState>((set) => ({
+	isExperimental: getLocalStorage('isExperimental') || false,
```
Comment (maxichrome) [inline packages/interface/src/components/device/Stores.tsx:13]: does this query local storage every time the value is accessed or only if setIsExperimental has been called?

**Review by maxichrome — APPROVED** [review]
Sweet, having this persisted will be super nice; just one comment to consider re: refetching from localStorage.

File: packages/interface/src/components/device/Stores.tsx (hunk 1 above)
Comment (xPolar) [reply packages/interface/src/components/device/Stores.tsx:13]: wdym

Comment (maxichrome) [reply packages/interface/src/components/device/Stores.tsx:13]: as in, does this call `getLocalStorage()` every time you access `useNodeStore().isExperimental`, or only when `create()` is called initially and when the setter is run?

Comment (xPolar) [reply packages/interface/src/components/device/Stores.tsx:13]: It does it every time.

Comment (maxichrome) [reply packages/interface/src/components/device/Stores.tsx:13]: Should be fine but we could revisit later for efficiency and all that good stuff

**Review by maxichrome — APPROVED** [review]
pog

**Pull request merged**